components/delete_post_modal.jsx

```diff
@@ -42,8 +39,8 @@ export default class DeletePostModal extends React.Component {
     componentDidUpdate(prevProps, prevState) {
         if (this.state.show && !prevState.show) {
             setTimeout(() => {
-                $(ReactDOM.findDOMNode(this.refs.deletePostBtn)).focus();
-            }, 0);
+                this.deletePostBtn.focus();
```

Comment:
Should we add conditional check here: 
```
if (this.deletePostBtn) {
    this.deletePostBtn.focus();
}
```

Reply:
Yeah, I can add that. :)

Reply:
Yep, I can add that. It is a great suggestion as the component could, theoretically, be unmounted before the setTimeout is finished.